lib/flash.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+const util_1 = require("util");
+function flashFactory() {
+    let currentSession;
+    let flashSession;
+    return {
+        request(type, ...message) {
+            if (message.length === 0) {
+                throw new Error('Provide a message to flash.');
+            }
+            if (Array.isArray(message[0])) {
+                message[0].forEach(msg => {
+                    ;
+                    (flashSession[type] = flashSession[type] || []).push(msg);
+                });
+                return flashSession[type].length;
+            }
+            console.log('message', message);
```

Comment:
this does not seem to match what we are compiling from.
Should we commit this?

@Ethan-Arrowood  what would you recommend?

Reply:
There is a problem with a automatic TS rebuild. The commited JS is old, i ll rebuild it and update the PR.

Reply:
Done.